lib/init.js

```diff
@@ -61,7 +61,7 @@ module.exports = async () => {
     console.log('')
     console.log(chalk.dim('This is a preview of the Testdriver.ai'));
     console.log(chalk.dim('Please report any issues in our Discord server: '));
-    console.log(chalk.dim('https://discord.gg/3Q3QkZ2'));
+    console.log(chalk.dim('https://discord.gg/22RdN6tp'));
```

Comment:
why keep changing?

Reply:
ya so some of these links are configured to expire after 7 days, I am just fixing the broken ones